Siri/L5/Level5.java

```diff
@@ -0,0 +1,105 @@
+                    System.out.println("     " + (i + 1) + "." + currTask);
+                }
+                System.out.println(line);
+            } else if (word.equals("mark")) { //marks the specified task with an "X"
+                int num = sc.nextInt();
+                int index = num - 1;
+                Task t = arrList.get(index);
+                t.markAsDone();
+                arrList.set(index, t);
+                System.out.println(line + "     This task has been marked:\n       "
+                        + t.toString() + "\n"
+                        + line);
+            } else if (word.equals("unmark")) { //unmarks the specified task
+                int num = sc.nextInt();
+                int index = num - 1;
+                Task t = arrList.get(index);
+                t.markAsNotDone();
+                arrList.set(index, t);
+                System.out.println(line + "     This task has been unmarked:\n       "
+                        + t.toString() + "\n"
+                        + line);
+            } else if (word.equals("todo")) { //creates todo task with "[T]" prefix and adds to the list
+                String desc = sc.nextLine();
+                try {
+                    checkRestOfInput(desc);
+                    Todo t = new Todo(desc);
+                    arrList.add(t);
+                    System.out.println(line + "     This task has been added to your list:\n       "
+                            + t.toString() + "\n" + "\n"
+                            + "     Number of task(s) in your list: " + arrList.size() + "\n"
+                            + line);
+                } catch (Exception e) {
+                    System.out.println(line + e.toString() + "\n" + line);
+                }
+            } else if (word.equals("deadline")) {
+                //creates deadline task with "[D]" prefix and deadline postfix and adds to list
+                while (sc.hasNextLine()) {
+                    String[] userInput = sc.nextLine().split("/by");
+                    // The userInput array now contains [taskDescription, deadline]
+                    Deadline deadLineTask = new Deadline(userInput[0], userInput[1]);
+                    arrList.add(deadLineTask);
+                    System.out.println(line + "     This task has been added to your list:\n       "
+                            + deadLineTask.toString() + "\n" + "\n"
+                            + "     Number of task(s) in your list: " + arrList.size() + "\n"
+                            + line);
+                    break;
+                }
+            } else if (word.equals("event")) {
+                //creates event task with "[E]" prefix and event date postfix and adds to list
+                while (sc.hasNextLine()) {
+                    String[] userInput = sc.nextLine().split("/at");
+                    // The userInput array now contains [taskDescription, event date]
```

Comment:
I like this comment, which is helpful for reviewers and compliant with the coding standard